app/controllers/api_controller.rb

```diff
@@ -10,4 +10,12 @@ def current_player
                         when current_game.player_2.api_key then current_game.player_2
                         end
   end
+
+  # def current_player?
+  #   request.headers['X-API-Key'] == current_player.api_key
+  # end
+
+  def current_player?
+    current_game.current_turn == 'player_1' && request.headers['X-API-KEY'] == current_game.player_1.api_key || current_game.current_turn == 'player_2' && request.headers['X-API-KEY'] == current_game.player_2.api_key
+  end
 end
```

Comment:
Here were the two current_player methods I was playing with, the one that is not commented will get us past the 49/72 test break that I kept arriving at with the first method.